src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.targets

```diff
@@ -37,7 +37,8 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 
   <PropertyGroup>
     <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' " >mscorlib</TargetProfile>
-    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' " >netcore</TargetProfile>
+    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' AND ($(TargetFramework.StartsWith(netcoreapp1.)) OR $(TargetFramework.StartsWith(netstandard1.)))" >netcore</TargetProfile>
```

Comment:
The TargetFramework property is an alias. One can have a project where TargetFramework=Foo but map it to some real targetframework (see https://github.com/dotnet/project-system/issues/1938#issuecomment-294324016 for an example of this). You could instead check that TargetFrameworkIdenitifier == .NETStandard\.NETCore and TargetFrameworkVersion > v1.0